yt/data_objects/grid_patch.py

```diff
@@ -414,7 +414,7 @@ def select(self, selector, source, dest, offset):
         else:
             slices = get_nodal_slices(source.shape, nodal_flag, dim)
             for i, sl in enumerate(slices):
-                dest[offset : offset + count, i] = source[sl][np.squeeze(mask)]
+                dest[offset : offset + count, i] = source[sl][tuple(np.squeeze(mask))]
```

Comment:
I think it should be `tuple()` around `sl` instead.

Reply:
oh right, good catch !

Reply:
fixed !